app/assets/purs/src/EntryPoints/Comments.purs

```diff
@@ -81,6 +113,20 @@ def props =
                             [ H.text comment.comment ]
                           , H.div "row"
                             [ H.text comment.commenter ] 
+                          , H.div "row"
+                            [ H.text $ "Upvotes:" <> Int.toStringAs Int.decimal comment.upvotes
```

Comment:
You can actually just use `show` here
```suggestion
                            [ H.text $ "Upvotes:" <> show comment.upvotes
```

Reply:
It took me an embarrassingly long time to get this to work 🤦 I knew there had to be an easy way like this but I couldn't find it